src/visitor.js

```diff
@@ -205,15 +194,27 @@ export default (code, out, options = {}) => {
       exit(path) {
         // Push all template literals leaded by graphql magic comment
         // e.g. /* GraphQL */ `query myQuery {}` -> query myQuery {}
+
+        if (!isValidMagicComment(path.node.leadingComments)) return
         if (!t.isTemplateLiteral(path.node.expression)) return
 
-        pluckMagicTemplateLiteral(path.node)
+        const gqlTemplateLiteral = pluckStringFromFile(path.node.expression)
```

Comment:
So basically the fixed was that you used `path.node.expression` and not `path.node`. Right? I don't see why you split the logic of the function. Just provide a second parameter.